ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PeerSimulator/Run.hs

```diff
@@ -97,6 +96,10 @@ data SchedulerConfig =
     -- | Whether to enable to LoP. The parameters of the LoP come from
     -- 'GenesisTest'.
     , scEnableLoP                :: Bool
+
+    -- | Enable node downtime if this is 'Just', using the value as minimum tick
+    -- duration to trigger it.
+    , scDowntime                 :: Maybe DiffTime
```

Comment:
Shouldn't we add disconnection explicitly in our schedules?

Reply:
But I guess this works well enough for this PR.

Reply:
we should, and I've sketched an alternative peers schedule generator that makes uptime intervals a first class concept (see my [design document](https://github.com/tweag/private-iog-genesis/issues/55)), but it's low priority for the remaining resources we have right now. I'll push my PoC code though

Reply:
Fair enough, yes. We could do that as part of the May milestone if we feel like it's worth it.